tests/Common/CommonSchemaTest.php

```diff
@@ -405,6 +405,23 @@ public function testGetTableNames(array $pdoAttributes): void
         $db->close();
     }
 
+    public function testHasTable(): void
+    {
+        $db = $this->getConnection(true);
+
+        $schema = $db->getSchema();
+
+        $this->assertTrue($schema->hasTable('customer'));
+        $this->assertTrue($schema->hasTable('category'));
+        $this->assertFalse($schema->hasTable('no_such_table'));
+
+        $db->createCommand()->dropTable('customer')->execute();
+
+        $this->assertFalse($schema->hasTable('customer', '', true));
```

Comment:
To fix tests try to use another table, without relations. Or create a new table for the test.

Reply:
New table worked jsut fine.